etc/cron.d/bedrock-prod.tmpl

```diff
@@ -9,6 +9,7 @@ MAILTO="webops-cron@mozilla.com,cron-bedrock@mozilla.com"
 
 */10 * * * * {{ user }} {{ source }}/bin/update-scripts/prod/update-prod-php.sh {{ log }}
 */15 * * * * {{ user }} {{ source }}/bin/update-scripts/prod/update-prod-locale-cron.sh {{ log }}
+0    * * * * {{ user }} {{ source }}/bin/update-scripts/prod/update-prod-product-details.sh {{ log }}
```

Comment:
I could, but I'd also need to reduce the cache timeout to 5 min as well or it'd be pointless. Seems a little short, but it would probably be okay.